### PR TITLE
First character data, fragments

### PR DESCRIPTION
**Pull request by przforme**

### CODE REVIEW

**Comment by przforme** [comment]
@veelenar Masz możliwość merge'owania? :v Zapomniałem dodać kogokolwiek do review i teraz chyba nie można zrobić merge xD

File: app/src/main/java/yacs5e/ptpthingers/com/yacs5e_app/LoginTest.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        @Override
+        protected String doInBackground(Void... nothing) {
+            try {
+                // creating comm channel
+                mChannel = ManagedChannelBuilder.forAddress(mHost, mPort).usePlaintext(true).build();
+                // creating blocking stub for sync
+                YACS5eGrpc.YACS5eBlockingStub stub = YACS5eGrpc.newBlockingStub(mChannel);
+
+                // creating message
+                TUser user = TUser.newBuilder().setLogin(username).setPassword(userpass).build();
```
Comment (krzysztofromanowski94) [inline app/src/main/java/yacs5e/ptpthingers/com/yacs5e_app/LoginTest.java]: Can we split that line? I feel like it's too many changes and it's loosing readability.

File: app/src/main/java/yacs5e/ptpthingers/com/yacs5e_app/SavingThrow.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    Byte disadvantage;
+
+    public SavingThrow() {
+        proficient = Boolean.FALSE;
+        advantage = 0;
+        disadvantage = 0;
+    }
+
+    public Boolean getProficient() {
+        return proficient;
```
Comment (krzysztofromanowski94) [inline app/src/main/java/yacs5e/ptpthingers/com/yacs5e_app/SavingThrow.java]: Proposition:
Maybe some function that takes a list as an argument and count the resulting advantage?

File: app/build.gradle [hunk 3]
```diff
@@ -17,18 +18,56 @@ android {
...
+    plugins {
+        javalite {
+            artifact = 'com.google.protobuf:protoc-gen-javalite:3.0.0'
+        }
+        grpc {
+            artifact = 'io.grpc:protoc-gen-grpc-java:1.7.0'
+        }
+    }
+    generateProtoTasks {
+        all()*.plugins {
```
Comment (krzysztofromanowski94) [inline app/build.gradle]: I remember that I added this entry, but you still came back with the two you previously wrote. Did you check that we need all of those three entries?

File: app/src/main/java/yacs5e/ptpthingers/com/yacs5e_app/AbilityScore.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
+package yacs5e.ptpthingers.com.yacs5e_app;
+
+class AbilityScore {
+    Integer value;
+    Integer modifier;
+    Integer tempValue;
+    Integer tempModifier;
+    Boolean useTemp;
+
+    public AbilityScore() {
```
Comment (krzysztofromanowski94) [inline app/src/main/java/yacs5e/ptpthingers/com/yacs5e_app/AbilityScore.java]: I think we should delete default constructor in future.

Comment (theamazingptp) [reply app/src/main/java/yacs5e/ptpthingers/com/yacs5e_app/AbilityScore.java]: Oh you're NUT!

File: app/src/main/java/yacs5e/ptpthingers/com/yacs5e_app/LoginTest.java (hunk 1 above)
Comment (theamazingptp) [reply app/src/main/java/yacs5e/ptpthingers/com/yacs5e_app/LoginTest.java]: Oh my good you're sooo pitchy.

File: app/src/main/java/yacs5e/ptpthingers/com/yacs5e_app/SavingThrow.java (hunk 2 above)
Comment (theamazingptp) [reply app/src/main/java/yacs5e/ptpthingers/com/yacs5e_app/SavingThrow.java]: I don't like it ;x

File: app/build.gradle (hunk 3 above)
Comment (theamazingptp) [reply app/build.gradle]: You added this entry? And who do you think you are!?

File: app/src/main/java/yacs5e/ptpthingers/com/yacs5e_app/LoginTest.java (hunk 1 above)
Comment (przforme) [reply app/src/main/java/yacs5e/ptpthingers/com/yacs5e_app/LoginTest.java]: "Picky" is not spelled like that.

**Pull request closed without merging**